docs/roadmap.md

```diff
@@ -0,0 +1,35 @@
+# Roadmap
+
+This is the official roadmap for the Open Service Broker for Azure (OSBA) project. OSBA is currently available as an alpha release and this roadmap represents our plan to deliver (OSBA) and the services it provides in a stable form.
+
+This roadmap addresses stability in terms of two dimensions:
+
+* Services/Plans
+* Database Schemas
+
+See below for more information concerning stability plans for each.
+
+# Services and Plans
+
+Services/Plans have three stability tiers:
+
+- `experimental` - We have an idea for a new service that we want to support, but don't have a good idea how it should look. We are essentially “throwing something against the wall” to see if it'll stick. Experimental services may be radically changed or removed at any time
+- `preview` - We have a better understanding how a service should look, but we don't yet guarantee backward compatibility. We do guarantee that this service won't go back to `experimental`, so we are committing to making it exist in some shape or form
```

Comment:
can we make a more specific guarantee here? for example, are we committing to working on it until it becomes stable in some shape or form?

Reply:
Not sure if we can commit to that in all cases. It's just occurred to me that some of the Azure-managed services that our modules can provision are _themselves_ in preview. For instance, Azure Database for MySQL || PostgreSQL are themselves both in preview. In such cases, we probably cannot make any guarantees.

Reply:
Can we commit to maturing the modules to match the maturity level of the service itself? I.e. we won't make Azure DB for MySQL or PostgreSQL stable until the underlying service is generally available and out of preview?